iac/deploy-stack.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
Have you checked to see if most of what this script provides can be done with serverless.com out of the box?

Reply:
I didn't find anything last time I checked but will check again now that I'm more familiar.

Reply:
I have a hard time finding anything related to serverless specifically. Everything I've found relates specifically to lamdba functions, and this stack doesn't have any lambdas.